### DISCUSSION TITLE
Anime Matrix gif fps is very low

### DISCUSSION THREAD

**Discussion by adityamn999** [seerge/g-helper#177] [Ideas]
All the GIFs I tried using were slowed down and looked choppy. Is there something I'm doing wrong? 
Also, is it possible to have the taskbar icon in black & white, in similar style to windows native ones? I love the app! Want to keep it docked beside wifi and volume. I can help out if needed with the design.

**Comment by seerge** [comment]
App intentionally limits matrix rendering to 10 frames per second to be resource savvy :) I may change it in future tho.

As for the icon - it's intentionally has green / blue / red colors depending on GPU mode. Lots of windows native apps has colorful icons as well (defender / teams / etc)
